docker/neo4j.yml

```diff
@@ -1,8 +1,8 @@
+version: "3.8"
 services:
   neo4j:
-    image: neo4j:4.3.7
+    image: neo4j:4.4.3
     hostname: neo4j
-    container_name: neo4j
```

Comment:
This was causing issues for me locally because I already have a built image named `neo4j`.